tests/helpers/participants.ts

```diff
@@ -168,8 +168,8 @@ async function joinTheModeratorAsP1(ctx: IContext, options?: IJoinOptions) {
     if (!options?.skipFirstModerator) {
         // we prioritize the access token when iframe is not used and private key is set,
         // otherwise if private key is not specified we use the access token if set
-        if (process.env.JWT_ACCESS_TOKEN && (!ctx.jwtPrivateKeyPath || !ctx.iframeAPI
-            || !options?.forceGenerateToken)) {
+        if (process.env.JWT_ACCESS_TOKEN
+            && (ctx.jwtPrivateKeyPath && !ctx.iframeAPI && !options?.forceGenerateToken)) {
```

Comment:
style: we don't need the braces anymore

Also this looks weird, we are going to use `process.env.JWT_ACCESS_TOKEN` only if we actually have set `ctx.jwtPrivateKeyPath`.  This will mean in practically we won't set `token` if  `ctx.jwtPrivateKeyPath` is `undefined`. I don't think this matches the comment...